andela/wger_asgardians
# #147322097 Enable Gym Members Comparison

#### What does this PR do?
Enable the comparison of any two members of the same gym while training together. This feature is available to and is used by the gym trainer, or administrator to have a visual indicator how any two members compare with each other in terms of height, weight, age, work/free hours, calory intakes etc. 
#### Description of Task to be completed?

1. Add access via a button on the gym overview to the user comparison page.
2. Ability to select any active user so that their data can be available on the graph. One can select two users at any one time, selecting a third deselects the earlier most selected member, and allows comparison of the last two.
3. Show the selected users' data side by side in the form of animated bar graphs.

#### How should this be manually tested?

1. Clone the repository.
2 . cd into the wger_argadianz directory and start the server via:
`python manage.py runserver`
3. Ensure presence of at least three gym members with their data updated.
4. On the gym overview page, click on the *Compare Members* button.
5. Click on any two members.
6. Their data should now be displayed on a graph to the left.
The graphs are colour-coded, and the key clearly indicates to whom the metrics belong to.
7. Selecting a third user should deselect one of the users, and pull up the now selected users's data.

#### Any background context you want to provide?
Currently there is no way to display this information.
#### What are the relevant pivotal tracker stories?
#147322097
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2017-07-14 at 15 13 20" src="https://user-images.githubusercontent.com/28535137/28212433-09165cb0-68aa-11e7-8449-ab9a4093f50f.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 06" src="https://user-images.githubusercontent.com/28535137/28212434-0918bffa-68aa-11e7-996f-825c875d97c0.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 29" src="https://user-images.githubusercontent.com/28535137/28212436-09195794-68aa-11e7-90b0-a52b75dea7ba.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 35" src="https://user-images.githubusercontent.com/28535137/28212432-09163e38-68aa-11e7-9d6b-28b8f7616d60.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 38" src="https://user-images.githubusercontent.com/28535137/28212437-0919a8f2-68aa-11e7-8899-0d69c0b430ff.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 42" src="https://user-images.githubusercontent.com/28535137/28212435-091911b2-68aa-11e7-810f-b3f6370b62b1.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 45" src="https://user-images.githubusercontent.com/28535137/28212438-093da2f2-68aa-11e7-9ecd-6834cf122410.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 48" src="https://user-images.githubusercontent.com/28535137/28212439-093ef9c2-68aa-11e7-83a6-2e3c5403e3fe.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 52" src="https://user-images.githubusercontent.com/28535137/28212440-0940c982-68aa-11e7-882f-692f68108a53.png">
<img width="1440" alt="screen shot 2017-07-14 at 15 13 56" src="https://user-images.githubusercontent.com/28535137/28212441-09426396-68aa-11e7-86ab-10ce26762791.png">
